tools/monitoring/main.go

```diff
@@ -17,15 +17,33 @@ limitations under the License.
 package main
 
 import (
+	"flag"
 	"fmt"
 	"log"
 	"net/http"
 	"os"
+
+	"io/ioutil"
 )
 
+var dbConfig DBConfig
+
 const yamlURL = "https://raw.githubusercontent.com/knative/test-infra/master/tools/monitoring/sample.yaml"
 
 func main() {
+	var err error
+
+	fDbUser := flag.String("monitoring-database-user", "",
+		"Text file containing the user name of to connect to the monitoring database")
+	fDbPass := flag.String("monitoring-database-password", "",
```

Comment:
do we need flags for this?  I thought these were part of secrets

Reply:
Removed flag and read the secrets file directly.